src/main/java/com/tvtracker/enterprise/dao/MediaEntryDAOStub.java

```diff
@@ -9,8 +9,8 @@
 
 @Component
 public class MediaEntryDAOStub implements  IMediaEntryDAO {
-    HashMap<Integer, MediaEntry> entriesByID = new HashMap<Integer, MediaEntry>();
-    HashMap<String, HashMap<Integer, MediaEntry>> entriesByUsername = new HashMap<String, HashMap<Integer, MediaEntry>>();
+    HashMap<Integer, MediaEntry> entriesByID = new HashMap<>();
```

Comment:
does not need to be explicitly defined twice. Works with just <>.

Reply:
We used to have to put generics on both sides... but not anymore.